crud/awards.ts

```diff
@@ -0,0 +1,99 @@
+import { Request, Response } from 'express';
```

Comment:
Don't define the prisma client again it's already defined in the config folder import that client only